src/main/features/testing-support/routes/create-claim-draft.ts

```diff
@@ -34,7 +37,7 @@ export default express.Router()
 
       const roles: string[] = await claimStoreClient.retrieveUserRoles(user)
 
-      if (roles && !roles.some(role => role.includes('cmc-new-features-consent'))) {
+      if (!featureToggles.isAutoEnrollIntoNewFeatureEnabled() && roles && !roles.some(role => role.includes('cmc-new-features-consent'))) {
```

Comment:
Here as well. It should be an async

Reply:
Acknowledged. Necessary changes are made as we have discussed.